auth/config/auth_config.yml

```diff
@@ -26,7 +26,7 @@ ldap_auth:
   # specify them here. Plain text password is read from the file.
   bind_dn: "uid=serviceaccount,ou=it,dc=example,dc=com"
   # Make sure you remove newlines and carriage returns from the password file.
-  bind_password_file: /tmp/ldap_password.txt.clean
+  bind_password_file: /config/ldap_password.txt
```

Comment:
there is a `start.sh` to fix this issue as work around already.

https://github.com/kwk/docker-registry-setup/blob/master/auth/start.sh#L8